src/App.tsx

```diff
@@ -1,26 +1,49 @@
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-import { Loader, TodoFilter, TodoList, TodoModal } from './components';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+import { TodoList } from './components';
+import { TodoFilter } from './components';
+import { TodoModal } from './components';
+import { Loader } from './components';
```

Comment:
Consider consolidating these imports from './components' into a single import statement to improve code readability and maintainability.